packages/orbit-components/src/Box/types.d.ts

```diff
@@ -35,15 +35,15 @@ export type ColorTokens =
   | "cloudNormalHover"
   | "cloudNormalActive"
   | "cloudDark"
-  | "inkLighter"
-  | "inkLighterHover"
-  | "inkLighterActive"
   | "inkLight"
   | "inkLightHover"
   | "inkLightActive"
   | "inkNormal"
   | "inkNormalHover"
   | "inkNormalActive"
+  | "inkDark"
```

Comment:
could you reuse the logic that I made recently for Separator? Have a typescript.template file with the type, that will allow us to have aligned with tokens and no need to update manually :)

Reply:
Had to do some "gymnastics" to avoid the breaking change, but I think it's ok 😅 🤸